test/lib/decentraland/lww-crdt-rules.spec.ts

```diff
@@ -22,18 +22,18 @@ describe('Conflict resolution rules for LWW-ElementSet based components', () =>
 
   it('PUT an unexistent value should succeed', () => {
     const entityId = 0 as Entity
+    const conflictBuffer = new ReadWriteByteBuffer()
 
-    const [conflict, currentValue] = updateFn({
+    const updateAccepted = updateFn({
```

Comment:
_Not sure / not very important_

This `updateAccepted` const here really clarifies the code. I don't have to remember what the update functions return. The same could be done in other places.

This may not be the best example because of the null-check, but `updateAccepted` is much clearer than:

```
if (component && component.updateFromCrdt(crdtMessage, this.outgoingMessagesBuffer)) {
```